pkg/apis/aadpodidentity/types.go

```diff
@@ -146,40 +180,41 @@ type AzureIdentitySpec struct {
 	Replicas *int32 `json:"replicas"`
 }
 
+// AzureIdentityStatus contains the replica status of the resource.
 type AzureIdentityStatus struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	AvailableReplicas int32 `json:"availableReplicas"`
 }
 
-/*** AzureIdentityBinding ***/
-type MatchType int
-
-const (
-	Explicit MatchType = 0
-	Selector MatchType = 1
-)
-
-//AssignedIDState -  State indicator for the AssignedIdentity
+// AssignedIDState represents the state of an AzureAssignedIdentity
 type AssignedIDState int
 
 const (
-	//Created - Default state of the assigned identity
+	// Created - Default state of the assigned identity
 	Created AssignedIDState = 0
-	//Assigned - When the underlying platform assignment of EMSI is complete
-	//the state moves to assigned
+
+	// Assigned - When the underlying platform assignment of EMSI is complete,
+	// the state moves to assigned
 	Assigned AssignedIDState = 1
 )
 
 const (
-	AzureIDResource                = "azureidentities"
-	AzureIDBindingResource         = "azureidentitybindings"
-	AzureAssignedIDResource        = "azureassignedidentities"
+	// AzureIDResource is the name of AzureIdentity.
+	AzureIDResource = "azureidentities"
+
+	// AzureIDBindingResource is the name of AzureIdentityBinding.
+	AzureIDBindingResource = "azureidentitybindings"
+
+	// AzureAssignedIDResource is the name of AzureAssignedIdentity.
+	AzureAssignedIDResource = "azureassignedidentities"
+
+	// AzureIdentityExceptionResource is the name of AzureIdentityException.
 	AzureIdentityExceptionResource = "azurepodidentityexceptions"
```

Comment:
Let's change this var to `AzurePodIdentityExceptionResource` to align with the other resource variables